noxfile.py

```diff
@@ -42,7 +42,8 @@ def test(session):
     session.install(".[test]")
 
     options = session.posargs
-    session.run('pytest', '--ignore', 'examples/', '-v', *options)
+    # -W=error raises pytest warnings to errors so they are caught by CI
```

Comment:
Nice solution, but are there any warning we don't care to raise as errors? If so, what do we do?

Reply:
we should resolve the warnings, but we can also ignore them. I'll put a link to the pytest docs to indicate how

Reply:
done!